js/miniatures.js

```diff
@@ -0,0 +1,22 @@
+import {createPhotos} from './data.js';
+
+const MAX_PHOTO_COUNT = 25;
+
+const otherPicturesList = document.querySelector('.pictures');
+const otherPicturesTemplate = document.querySelector('#picture')
+  .content
+  .querySelector('.picture');
+const otherPictures = createPhotos(MAX_PHOTO_COUNT);
+const otherPicturesFragment = document.createDocumentFragment();
```

Comment:
Так как фрагмент в модуле один то можно просто 
```suggestion
const otherPicturesFragment = document.createDocumentFragment();
```
```suggestion
const fragment = document.createDocumentFragment();
```